examples/example_patchextraction.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>VariableWindow is not implemented yet so I would remove it for now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>